integration/querydsl/testsuite/src/test/java/com/blazebit/persistence/querydsl/BasicQueryTest.java

```diff
@@ -243,7 +244,7 @@ public void testModuloLongOperation() {
     }
 
     public static BooleanExpression bitwiseAndLong(NumberExpression<Long> expression, long bit) {
-        NumberExpression<Long> mod = expression.divide(1L << bit).floor().mod(2L);
+        NumberExpression<Long> mod = castToNum(Long.class, expression.divide(1L << bit).floor()).mod(2L);
```

Comment:
From the remaining test failure, it appears as if MySQL does not allow `cast(x as bigint)`. I believe I am using `CAST_LONG` from the [cast and treat functions](https://persistence.blazebit.com/documentation/1.6/core/manual/en_US/index.html#cast-and-treat-functions). Shall I just exclude the test for one or the other dialect?

Reply:
According to MySQL docs, `bigint` should have been fine: https://dev.mysql.com/doc/refman/8.0/en/integer-types.html